### ISSUE TITLE
Thermostat dont start 

### ISSUE THREAD

**Issue by yuripace** [nielsfaber/scheduler-card#233]
I explain my situation.
I have a switch i use for start my heating manually, and a thermostat linked to that switch.
I scheduled the switch to start and stop in different time of the day. When the switch is off, i want to leave running my thermostat (i want to leave heating running sometime without considering temperature), but seems that the scheduled switch "won" on the thermostat and the thermostat dont start.
I'm making something wrong or?

**Comment by nielsfaber** [comment]
I don't really know what you're making.
Sounds like you manually turn on/turn off a switch to control the temperature.
In this case you should use the generic_thermostat implementation in HA.

**Comment by yuripace** [comment]
Yes, exactly. I use both of them, I have the thermostat but i can start manually my heating. But the scheduling enabled on the switch "won" on the thermostat implementation in HA. This is the problem.
THe manual switch allow me to start heating even if i have 30° in my house

**Comment by KTibow** [comment]
So you're saying that you want there to be an option to disable a schedule if someone manually controls the entity?

**Comment by yuripace** [comment]
I dont know if it's the right choice..an action like "leave untouched" may be better? So in a certain time during the schedule, the entity can be controlled manually (or by some other HA entities).

**Comment by KTibow** [comment]
Could you post a screenshot of your schedule right now?

**Comment by derekoharrow** [comment]
I've seen something similar described in the schedy hass app (which is VERY complicated in my view) - https://hass-apps.readthedocs.io/en/latest/apps/schedy/tutorial.html#automatic-re-scheduling-after-manual-adjustments

Basically, you have the option to specify a duration during which manually made changes are not over-written by the schedule.
For example, I have a schedule that sets my underfloor heating to 23C during the evening. If I came along and manually changes that down to 20C, the scheduler would leave it alone for the specified period (e.g. 30 mins) before changing it back again.

Another example - I have a script to dim some lights at 23:30, but if I've already turned those lights off, then the scheduler will actually turn them back on again, so again, if they've been manually adjusted during a set time, don't change them.
